GoogleForCreators/web-stories-wp
# Allow publishing stories "Immediately"

## Context
Previously the creation date of the story was used by default for publishing.


## Summary
The default date for publishing is now the date when the story gets published unless it's specifically set for something else.


## Relevant Technical Choices



## To-do

- [x] Verify tests still work.



## User-facing changes
![publish](https://user-images.githubusercontent.com/3294597/119828667-b011f380-befa-11eb-8fb8-941b8ef71ffc.gif)



## Testing Instructions
1. Add a new story and open the Document panel
2. Verify the publish date shows "Immediately"
3. Now change the date to something else, verify the date gets changed in the input, too.
4. Open the calendar again and click the "Reset" button in the bottom.
5. Verify the date shows "Immediately" again.
6. Set a new date again, e.g. something in the past. Now publish the story.
7. Refresh the editor (just to ensure everything got saved correctly). Verify the publish date in the publish field displays the date previously chosen.


### QA






This PR can be tested by following these steps:

1. Follow the steps above under testing instructions.

### UAT







This PR can be tested by following these steps:

1.

## Checklist




---



Fixes #7692
